src/cursor/cur_backup_incr.c

```diff
@@ -202,6 +205,10 @@ __curbackup_incr_next(WT_CURSOR *cursor)
             if (__bit_test(cb->bitstring.mem, cb->bit_offset)) {
                 found = true;
                 WT_STAT_CONN_INCR(session, backup_blocks);
+                if (F_ISSET(cb, WT_CURBACKUP_COMPRESSED))
+                    WT_STAT_CONN_INCR(session, backup_blocks_compressed);
+                else
+                    WT_STAT_CONN_INCR(session, backup_blocks_uncompressed);
```

Comment:
Should these two be `WT_STAT_CONN_DATA_INCR()`?

Reply:
Refresh. I caught that and fixed.

Reply:
Thanks Sue! And there's a test to go with it. Nice. 

My previous LGTM still stands :-)